resource-timing/initiator-type/link.html

```diff
@@ -30,6 +31,7 @@
   initiator_type_test("resource_timing_test0.css?id=preload", "link", "<link preload>");
   initiator_type_test("green.html?id=prerender", "link", "<link prerender>");
   initiator_type_test("manifest.json", "link", "<link manifest>");
+  initiator_type_test("resources/empty.js?id=modulePreload", "other", "module preload");
```

Comment:
Why "other"?

Reply:
That's a very good question. With initiator types, for now I'm spec'ing them based on what implementations currently do. I think once we have all the specs in place that reflect the status quo we should do an initiator type overhaul. Right now they don't serve their purpose as they show "other" exactly in the places where you'd want them to be meaningful.

Reply:
That's fair. Can you add a comment indicating that one would expect this to be "link", but this is what implementations are currently doing?